com.github.IsmaelMartinez.teams_for_linux.appdata.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.IsmaelMartinez.teams_for_linux</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>teams-for-linux</name>
+  <summary>Unofficial Microsoft Teams client for Linux using Electron</summary>
+  <description>
+    <p>Unofficial Microsoft Teams client for Linux using Electron.</p> 
+    <p>It uses the Web App and wraps it as a standalone application using Electron.</p>
+  </description>
+  <categories>
```

Comment:
The categories should be put on the desktop file. They are merged with the appdata file later.